ftl/python/layer_builder.py

```diff
@@ -39,26 +39,14 @@ def __init__(self,
         self._cache = cache
 
     def GetCacheKeyRaw(self):
-        descriptor_contents = ftl_util.descriptor_parser(
-            self._descriptor_files, self._ctx)
-        return '%s %s' % (descriptor_contents,
-                          self._dep_img_lyr.GetCacheKeyRaw())
+        return ""
 
     def BuildLayer(self):
-        cached_img = None
+        with ftl_util.Timing('Building pkg layer'):
+            self._build_layer()
         if self._cache:
-            with ftl_util.Timing('Checking cached pkg layer'):
-                key = self.GetCacheKey()
-                cached_img = self._cache.Get(key)
-                self._log_cache_result(False if cached_img is None else True)
-        if cached_img:
-            self.SetImage(cached_img)
-        else:
-            with ftl_util.Timing('Building pkg layer'):
-                self._build_layer()
-            if self._cache:
-                with ftl_util.Timing('Uploading pkg layer'):
-                    self._cache.Set(self.GetCacheKey(), self.GetImage())
+            with ftl_util.Timing('Uploading pkg layer'):
+                self._cache.Set(self.GetCacheKey(), self.GetImage())
```

Comment:
I'm a little confused here. Looks like we're not checking the cache for these layers anymore? Is that intentional?

Reply:
yes.  we only upload these layers to get faster layer caching on the GCR side.  FTL doesn't use these uploaded layers directly

Reply:
shouldn't we at least check to see if we've already uploaded this layer so we don't build it again?

Reply:
the question here is what the cache key is.  the only way to know if we'd already built it would be to build it at this point.  this is a consequence of requirements.txt not being specified

Reply:
we can do what you are talking about for Pipfile